ui/src/components/forms/Form.js

```diff
@@ -203,6 +208,7 @@ const Form = (props) => {
           field = (
             <Stack direction={'column'}>
               <Stack direction={['column', 'row']} w="100%">
+                s{' '}
```

Comment:
what is this supposed to be? seems like a typo

Reply:
yeah this is a typo